.github/workflows/create-release.yml

```diff
@@ -14,62 +14,71 @@ on:
-        type: boolean
-        default: false
+
+concurrency:
+  # make publishing release concurrent (but others trigger not)
+  group: building-releases-${{ inputs.publish-release && 'prerelease' || github.run_id }}
 
 jobs:
   configure:
     runs-on: ubuntu-latest
     outputs:
-      version: ${{steps.config.outputs.version}}.${{ steps.config.outputs.build_number}}
-      version_base: ${{steps.config.outputs.version}}
-      tag_name: v${{steps.config.outputs.version}}.${{ steps.config.outputs.build_number }}
-      release: v${{steps.config.outputs.version}}.${{ steps.config.outputs.build_number }}
-      changes: ${{ steps.config.outputs.changes }}
+      version: ${{ steps.read-version.outputs.version_full }}
+      version_base: ${{ steps.read-version.outputs.version }}
+      tag_name: v${{ steps.read-version.outputs.version_full }}
+      version_commit: ${{ steps.version_commit.outputs.commit_long_sha }}
+      pushed: ${{ steps.version_commit.outputs.pushed }}
+      tag_pushed: ${{ steps.version_commit.outputs.tag_pushed }}
     if: github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository == 'quarto-dev/quarto-cli')
     steps:
-      - name: Install libc6-div
-        run: sudo apt-get install libc6-dev
-
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: config
-        id: config
+      - name: Get previous version
+        id: read-version
         run: |
-          source ./configuration
-
-          CHANGES=
-          # CHANGES=$(git log $(git describe --tags --abbrev=0)..HEAD --oneline)
-          # Escape \n, \r to preserve multiline variable
-          # See https://github.community/t/set-output-truncates-multiline-strings/16852/2
-          # CHANGES="${CHANGES//'%'/'%25'}"
-          # CHANGES="${CHANGES//$'\n'/'%0A'}"
-          # CHANGES="${CHANGES//$'\r'/'%0D'}"
-          # echo "changes=$CHANGES" >> $GITHUB_OUTPUT
-
-          QUARTO_BUILD_NUMBER=$(($QUARTO_BUILD_RUN_OFFSET + $GITHUB_RUN_NUMBER))
-
-          echo "version=$QUARTO_VERSION" >> $GITHUB_OUTPUT
-          echo "changes=$CHANGES" >> $GITHUB_OUTPUT
-          echo "build_number=$QUARTO_BUILD_NUMBER" >> $GITHUB_OUTPUT
+          read_version=$(cat version.txt)
+          version=$(echo $read_version | grep -o '[[:digit:]]*\.[[:digit:]]*' | head -1)
+          build_number=$(($(echo $read_version | grep -o '[[:digit:]]*' | tail -1)+1))
+          version_full=$version.$build_number
+          echo "version=$version" >> $GITHUB_OUTPUT
+          echo "version_full=$version_full">> $GITHUB_OUTPUT
+          echo "**Changelog since last release**: https://github.com/${GITHUB_REPOSITORY}/compare/v${read_version}...v${version_full}" > "release_note.md"
+          echo -e "\n\nFull ${version} changelog up to this version:\n" >> "release_note.md"
+          echo -e "  * View: https://github.com/cderv/quarto-cli/blob/v${version_full}/news/changelog-${version}.md\n" >> "release_note.md"
```

Comment:
Shouldn't this line be: 

```
https://github.com/${GITHUB_REPOSITORY}/blob/v${version_full}/news/changelog-${version}.md\n" >> "release_note.md"
```

Reply:
yes it should have been but was missed. I was off for evening time but check back the first release, and already fixed at bb03084 

Thanks for watching out and the feedback !